src/Microsoft.Health.Dicom.SqlServer/Features/ExtendedQueryTag/SqlExtendedQueryTagStore.cs

```diff
@@ -43,7 +43,7 @@ public class SqlExtendedQueryTagStore : IExtendedQueryTagStore
             _logger = logger;
         }
 
-        public async Task AddExtendedQueryTagsAsync(IEnumerable<ExtendedQueryTagEntry> extendedQueryTagEntries, CancellationToken cancellationToken = default)
+        public async Task AddExtendedQueryTagsAsync(IEnumerable<ExtendedQueryTagStoreEntry> extendedQueryTagEntries, CancellationToken cancellationToken = default)
```

Comment:
Should input be _IEnumerable<AddExtendedQueryTagEntry>_ ? Since ExtendedQueryTagStoreEntry.Key, Status are not used by this method

Reply:
Sure!